notebooks/ray-examples/ray-torch.ipynb

```diff
@@ -0,0 +1,455 @@
+{
+ "cells": [
```

Comment:
pytorch usually has a dataloader class for creating batches and padding etc for the Neural network, I'm curious how that would play out with ray datasets...